### PR TITLE
Update/aad id v1.0 getowneditems

### PR DESCRIPTION
**Pull request by lleonard-msft**
Documenting v1.0 version of this endpoint, per Neehar.

@dkershaw10, please review.

### CODE REVIEW

File: misc/SEOKeys-en-us.xml [hunk 1]
```diff
@@ -5128,7 +5129,7 @@
         <Item text="List deleted items" url="/en-us/api-reference/beta/api/directory_deleteditems_list" />
         <Item text="Permanently delete item" url="/en-us/api-reference/beta/api/directory_deleteditems_delete" />
         <Item text="Restore deleted item" url="/en-us/api-reference/beta/api/directory_deleteditems_restore" />
-		<Item text="List deleted items owner by user" url="/en-us/api-reference/beta/api/directory_deleteditems_user_owned" />
+        <Item text="List deleted items owner by user" url="/en-us/api-reference/beta/api/directory_deleteditems_user_owned" />
```
Comment (dkershaw10) [inline misc/SEOKeys-en-us.xml]: This needs to be fixed in beta - List deleted items **owned** by user (not **owner**).  Same issue for v1.0 above.

File: misc/SEOKeys-en-us.xml [hunk 2]
```diff
@@ -1688,6 +1688,7 @@
       <Item text="List deleted items" url="/en-us/api-reference/v1.0/api/directory_deleteditems_list" />
       <Item text="Permanently delete item" url="/en-us/api-reference/v1.0/api/directory_deleteditems_delete" />
       <Item text="Restore deleted item" url="/en-us/api-reference/v1.0/api/directory_deleteditems_restore" />
+      <Item text="List deleted items owner by user" url="/en-us/api-reference/v1.0/api/directory_deleteditems_user_owned" />
```
Comment (dkershaw10) [inline misc/SEOKeys-en-us.xml]: items owner -> items owned

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
+# **List deleted items owned by a user**
+
+Retrieves a list of recently deleted items owned by the specified user.  
+
+Currently, delete items functionality is supported only for [group](../resources/group.md) resources owned by the user.
```
Comment (dkershaw10) [inline api-reference/v1.0/api/directory_deleteditems_user_owned.md]: delete items -> list deleted items

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+This is a service action, which means it does not support pagination.  The API returns up to 1,000 deleted objects owned by the user, sorted by ID.
```
Comment (dkershaw10) [inline api-reference/v1.0/api/directory_deleteditems_user_owned.md]: ID -> id.

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+POST /directory/deletedItems/getUserOwnedObjects
+```
+
+## Request headers
+
+| **Name**      | **Description**           |
+| ------------- | ------------------------- |
+| Authorization | Bearer {token}. Required. |
+
+## Request body
```
Comment (dkershaw10) [inline api-reference/v1.0/api/directory_deleteditems_user_owned.md]: For actions, don't we normally have a table describing the parameters in this section?
See https://developer.microsoft.com/en-us/graph/docs/api-reference/beta/api/directoryobject_checkmembergroups

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+Here is an example of the request.
+
+``` http
+POST https://graph.microsoft.com/v1.0/directory/deletedItems/getUserOwnedObjects
+Content-type: application/json
+```
+
+``` json
+{
+  "userId":"{id}",
```
Comment (dkershaw10) [inline api-reference/v1.0/api/directory_deleteditems_user_owned.md]: Can we put a random GUID in here?  Not sure if this works with a UPN too.

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+``` http
+HTTP/1.1 200
+Content-type: application/json
+Content-length: 1249
+
+{
+"value": [
+          {
+              "@odata.type": "#microsoft.graph.group",
+              "id": "",
```
Comment (dkershaw10) [inline api-reference/v1.0/api/directory_deleteditems_user_owned.md]: Stick a random GUID here?

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+              "deletedDateTime": null,
```
Comment (dkershaw10) [inline api-reference/v1.0/api/directory_deleteditems_user_owned.md]: If this is a deletedItem this property won't be null.
```json
{
    "@odata.context": "https://graph.microsoft.com/beta/$metadata#directoryObjects/$entity",
    "value": [
        {
            "odata.type": "Microsoft.DirectoryServices.Group",
            "id": "a7f54e51-21d8-47b9-87f3-7dacd4a20640",
            "deletedDateTime": "2018-06-08T01:01:05Z"
   }
  ]
}

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+| **Name**      | **Description**           |
+| ------------- | ------------------------- |
+| Authorization | Bearer {token}. Required. |
+
+## Request body
+
+```json
+{
+  "userId":"{id}",
+  "type":"group"
```
Comment (dkershaw10) [inline api-reference/v1.0/api/directory_deleteditems_user_owned.md]: This is wrong.  Valid value is "Group" (capitalized).

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md (hunk 6 above)
Comment (dkershaw10) [reply api-reference/v1.0/api/directory_deleteditems_user_owned.md]: Based on thread with Daron, I don't feel so strongly about this.  You can leave this as is.

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md (hunk 9 above)
Comment (lleonard-msft) [reply api-reference/v1.0/api/directory_deleteditems_user_owned.md]: Fixed

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md (hunk 8 above)
Comment (lleonard-msft) [reply api-reference/v1.0/api/directory_deleteditems_user_owned.md]: Fixed

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md (hunk 7 above)
Comment (lleonard-msft) [reply api-reference/v1.0/api/directory_deleteditems_user_owned.md]: Fixed

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md (hunk 6 above)
Comment (lleonard-msft) [reply api-reference/v1.0/api/directory_deleteditems_user_owned.md]: Fixed.

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md (hunk 5 above)
Comment (lleonard-msft) [reply api-reference/v1.0/api/directory_deleteditems_user_owned.md]: Fixed.

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md (hunk 4 above)
Comment (lleonard-msft) [reply api-reference/v1.0/api/directory_deleteditems_user_owned.md]: STET, per style guide.

File: api-reference/v1.0/api/directory_deleteditems_user_owned.md (hunk 3 above)
Comment (lleonard-msft) [reply api-reference/v1.0/api/directory_deleteditems_user_owned.md]: Fixed.

File: misc/SEOKeys-en-us.xml (hunk 2 above)
Comment (lleonard-msft) [reply misc/SEOKeys-en-us.xml]: Fixed

File: misc/SEOKeys-en-us.xml (hunk 1 above)
Comment (lleonard-msft) [reply misc/SEOKeys-en-us.xml]: Fixed.

**Comment by lleonard-msft** [comment]
@dkershaw10 - Changes incorporated; PTAL

File: api-reference/beta/api/directory_deleteditems_user_owned.md [hunk 10]
```diff
@@ -34,14 +34,21 @@ POST /directory/deletedItems/getUserOwnedObjects
 
 ```json
 {
-  "userId":"{id}",
+  "userId":"55ac777c-109e-4022-b58c-470c8fcb6892",
   "type":"group"
 }
 ```
 
+The request body supports the following optional parameters:
```
Comment (dkershaw10) [inline api-reference/beta/api/directory_deleteditems_user_owned.md]: These parameters are required.

File: api-reference/beta/api/directory_deleteditems_user_owned.md [hunk 11]
```diff
@@ -75,8 +82,8 @@ Content-length: 1249
 "value": [
           {
               "@odata.type": "#microsoft.graph.group",
-              "id": "",
-              "deletedDateTime": null,
+              "id": "55ac777c-109e-4022-b58c-470c8fcb6892",
```
Comment (dkershaw10) [inline api-reference/beta/api/directory_deleteditems_user_owned.md]: This GUID cannot be the same as the userId above.  The one above is for user.  This returned group cannot have the same id...

File: api-reference/beta/api/directory_deleteditems_user_owned.md (hunk 10 above)
Comment (lleonard-msft) [reply api-reference/beta/api/directory_deleteditems_user_owned.md]: Fixed.

File: api-reference/beta/api/directory_deleteditems_user_owned.md (hunk 11 above)
Comment (lleonard-msft) [reply api-reference/beta/api/directory_deleteditems_user_owned.md]: Fixed.

**Comment by lleonard-msft** [comment]
@dkershaw10 - Latest feedback applied; PTAL when the checks are completed.

**Comment by dkershaw10** [comment]
@lleonard-msft LGTM, merge!

**Pull request merged**